mdsf-vscode/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,46 @@
+# Welcome to your VS Code Extension
```

Comment:
I'm unsure about this one

Either, it could be removed, updated, or left like this.

Reply:
Moved to #649